docs/requester_pays.rst

```diff
@@ -0,0 +1,195 @@
+=======================================================================================
+Accessing Public Cloud Data with Requester Pays
+=======================================================================================
+
+The data we're publishing in the PUDL Catalog is publicly accessible and distributed
+under the permissive `CC-BY-4.0 <https://creativecommons.org/licenses/by/4.0>`__
+license. Catalyst covers the cost of storing the data in Google cloud storage buckets.
+However, there are also fees incurred when data leaves the Google Cloud Platform (GCP).
+Depending where you're downloading from, it costs $0.10-0.20 (USD) per GB.  In order to
+share large amounts of public data without risking unexpectedly large data egress
+charges due to someone maliciously or accidentally downloading a large volume of data,
+we've set our cloud storage to use `requester pays
+<https://cloud.google.com/storage/docs/requester-pays>`__.
+
+"Requester pays" means the person downloading the data is responsible for those costs
+instead. As a user paying the data egress charges associated with your data usage is
+cheap, but as a data provider covering the data egress charges for all of your users can
+quickly get expensive! Downloading all of the EPA CEMS, FERC 1, PUDL, and US Census data
```

Comment:
Not meaning to be nit picky about grammar and I also don't have the best grammar myself, but maybe would throw some commas in here for readability? 

"As a user, paying... is cheap, but as a data provider, covering..

Reply:
Simplified these sentences.